praw/config.py

```diff
@@ -139,3 +139,7 @@ def _initialize_attributes(self):
 
         for required_attribute in ("oauth_url", "reddit_url"):
             setattr(self, required_attribute, self._fetch(required_attribute))
+
+        self.validate_on_submit = self._config_boolean(
+            self._fetch_or_not_set("validate_on_submit")
+        )
```

Comment:
Can you make a loop like the one above on lines 129 but for `_config_boolean` and merge with `check_for_updates`?